src/webhooks/helius/helius.service.ts

```diff
@@ -102,10 +118,32 @@ export class HeliusService {
           itemsMinted: { increment: 1 },
         },
       });
+    const nftTransactionInfo = enrichedTransaction.events.nft as NFTEvent & {
+      amount: number;
+    };
+    const logIntoMintReceipt = this.prisma.mintReceipt.create({
+      data: {
+        buyer: nftTransactionInfo.buyer,
+        price: nftTransactionInfo.amount / LAMPORTS_PER_SOL, // not sure if there is an existing helper function that can convert to SOL properly (max 9 decimals I assume?)
+        mintedAt: new Date(nftTransactionInfo.timestamp),
+        description: enrichedTransaction.description,
+      },
+    });
 
-    await this.prisma.$transaction([
-      createComicIssueNft,
-      updateComicIssueCandyMachine,
-    ]);
+    try {
+      const [comicIssueNft] = await this.prisma.$transaction([
+        createComicIssueNft,
+        updateComicIssueCandyMachine,
+        logIntoMintReceipt,
```

Comment:
Let's skip using prisma.$transaction for now. If one of these 3 queries fail, it should log an error but we should still try executing remaining queries.

try catch each of these 3 queries and ditch the $transaction